modules/BTBidAdapter.js

```diff
@@ -31,7 +31,7 @@ function imp(buildImp, bidRequest, context) {
 
   if (params) {
     const { siteId, ab, ...btBidParams } = params;
```

Comment:
do we still need `siteId` and `ab` on line 33? Is the idea to exclude them from `imp.ext` or are they no longer in `params`?

Reply:
The idea is to send only `btBidParams` in `imp.ext`. On line 33 we exclude`siteId, ab` from all the other bidders params and send them in `site.ext.blockthrough`

Reply:
Gotcha, thanks for the explanation!